Automattic/jetpack
# Remove toolbar 'edit' control from the Contact Form block

Fixes #26867

#### Changes proposed in this Pull Request:

This patch removes the 'Edit' dropdown from the Contact Form block's toolbar, which was just a duplicate of the settings already contained in the sidebar - which is the expected place for such settings in Gutenberg anyway.  
This helps un-clutter the block UI.

Before:
![Screenshot 2022-10-18 at 18 51 01](https://user-images.githubusercontent.com/8056203/196498561-68f75954-242d-4262-82c7-e9a1bb5d98ae.png)

After:
![Screenshot 2022-10-18 at 18 50 44](https://user-images.githubusercontent.com/8056203/196498553-cf5f71f7-68ce-4d78-b1fc-b7c663e4adca.png)

#### Other information:


#### Jetpack product discussion

N/A

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:






- Checkout the branch and start/edit a post or a page in the block editor.
- Insert a Contact Form block.
- Make sure the Contact Form block is focused (clicking on the block once will focus the Contact Form block, subsequent clicks may focus child blocks).
- There should be no 'Edit' option in the block toolbar.
